lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -212,10 +212,12 @@
         },
 
         initialize: function( obj ) {
+            var el = $('#incourse-reverify-container');
```

Comment:
Nitpick: can you please follow the naming convention of leading with a $ for DOM elements (i.e. rename the var $el).